src/main/resources/templates/category.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org/" lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <meta charset="UTF-8">
+    <title>Shoe Review</title>
```

Comment:
Do you think this should be called `Category` or something?

Reply:
yes, it should be called that.

Reply:
Fixed in https://github.com/2020-Spring-Cohort/reviews-site-models/pull/37/commits/f4db108cdc1536f46444657d7d087d47186ad20a